setup.py

```diff
@@ -17,26 +18,34 @@
 
 SHELLCHECK_VERSION = '0.8.0'
 POSTFIX_SHA256 = {
-    # TODO(rhee): detect "linux.aarch64" and "linux.armv6hf"
-    'linux': (
+    # TODO: detect "linux.armv6hf"
+    ('linux', 'aarch64'): (
+        'linux.aarch64.tar.xz',
+        'a12bdfe0f95811ad6c0a091006b919b2834b0619b460cfa596f557edd62e45ab',
+    ),
+    ('linux', 'x86_64'): (
         'linux.x86_64.tar.xz',
         'ab6ee1b178f014d1b86d1e24da20d1139656c8b0ed34d2867fbb834dad02bf0a',
     ),
-    'darwin': (
+    ('darwin', 'x86_64'): (
         'darwin.x86_64.tar.xz',
         'e065d4afb2620cc8c1d420a9b3e6243c84ff1a693c1ff0e38f279c8f31e86634',
     ),
-    'win32': (
+    ('darwin', 'arm64'): (
+        'darwin.x86_64.tar.xz',
+        '969bd7ef668e8167cfbba569fb9f4a0b2fc1c4021f87032b6a0b0e525fb77369',
+    ),
```

Comment:
since this is identical it would be best to do this like the `cygwin` example below

Reply:
Thanks for suggestion. I have pushed the changes again.